package.json

```diff
@@ -42,6 +42,7 @@
     "typescript": "^4.7.3"
   },
   "devDependencies": {
+    "@arbitrum/sdk": "^3.1.2",
```

Comment:
seems like the dependency is used in the main entrypoint (`index.ts`) so this is not a devDependency